stackit/internal/services/iaas/publicip/resource_test.go

```diff
@@ -130,12 +136,14 @@ func TestToCreatePayload(t *testing.T) {
 				Labels: types.MapValueMust(types.StringType, map[string]attr.Value{
```

Comment:
you should add a test case here where the field is not set in the Model and check that in the payload it is sent as  nullable string with value nil

Reply:
Added a new test case `network_interface_id_nil`